src/Mentat.UI/Views/Home/Index.cshtml

```diff
@@ -1,12 +1,59 @@
-﻿@{
+﻿@model Mentat.UI.ViewModels.CardViewModel
+
+@{
     ViewData["Title"] = "Home";
 }
 
-<div class="container d-flex vh-60">
+<div class="container">
     <span class="dot"></span>
+    
+    <div class="container pt-5">
+        <div id="myCarousel">
+            <div id="carouselCard" class="carousel slide" data-ride="carousel">
+
+              <!-- Card Indicator -->
+              <ul class="carousel-indicators">
+                @{int j = 0;}
+                @foreach (var item in Model.cards)
```

Comment:
Can you do a regular for-loop? Applies to both for-eaches

Reply:
Out of curiosity, is there something wrong with a for-each-loop?

Reply:
Declaring an index outside a foreach is typically a hint you should be using a classic for loop. You've elected to use a List to hold your cards, therefore you have access to Count and index accessing. A for-loop is a better fit.